_visual/typography.md

```diff
@@ -75,7 +75,10 @@ order: 01
             <p><span class="usa-label-big">Fast</span></p>
             <p>Ideal number of fonts. Will allow for optimal page load performance.</p>
             <h6 class="usa-heading-alt">Example</h6>
-            <p><a href="{{ site.baseurl }}/assets-styleguide/img/epa-emanifest-screenshot.png">EPA eManifest  (screenshot of non-public site)</a></p>
+            <p>
```

Comment:
@maya they were in p tags for the links that were merged before so I just added the images. It gives consistent spacing so I left it in. Let me know if you think we should take it out.